server/src/index.ts

```diff
@@ -1,29 +1,103 @@
-import { redisGet, redisSet } from "./redis.ts";
-
 import { serve } from "https://deno.land/std@0.182.0/http/server.ts";
 
+import * as jose from "https://deno.land/x/jose@v4.14.1/index.ts";
+
+import { redisGet, redisSet, redisClient } from "./redis.ts";
+import { app, authenticated, Response404 } from "./framework.ts";
+
 const oneWeekInSeconds = 7 * 24 * 60 * 60;
 
+const auth0Tenant = "https://dev-gy4q5ggc5zaobhym.us.auth0.com/";
+const Auth0JKWS = jose.createRemoteJWKSet(
+  new URL(`${auth0Tenant}.well-known/jwks.json`)
+);
+
 serve(
-  async (request) => {
-    if (request.method === "POST") {
-      const { method, params } = await request.json();
-      if (method === "get") {
-        const { key, id } = params;
-        return new Response((await redisGet(id + key)) || "null");
-      }
-      if (method === "set") {
-        const { key, value, id, ttl } = params;
-        const ttlOrDefault = ttl || oneWeekInSeconds;
-        await redisSet(
-          id + key,
-          value,
-          ttlOrDefault > oneWeekInSeconds ? oneWeekInSeconds : ttlOrDefault,
-        );
-        return new Response(JSON.stringify({}));
-      }
-    }
-    return new Response();
-  },
-  { port: parseInt(Deno.env.get("PORT") as string) },
+  app({
+    "/": {
+      POST: authenticated(api_token)(async (request, auth) => {
+        const { method, params } = await request.json();
+        if (method === "get") {
```

Comment:
for consistency, let's choose between `==` and `===` and use it always

Reply:
Replaced all comparisons to strings or numbers with `===`. Comparisons to nulls are still not strict.